tmk_core/common/chibios/timer.c

```diff
@@ -1,4 +1,5 @@
 #include <ch.h>
+#include <hal.h>
```

Comment:
None of this should be introduced to timer.c, and should go in wait.c instead

Reply:
I've had to move `wait.c` into `wait.h` because it was being included from there.

I've also made an empty avr `wait.c` so that it still builds.

Reply:
Doesn't that require LTO?

Reply:
The problem is that it needs to be visible from wait.h to be inlined at the point of use.

Reply:
I could move it to a new `_wait.c`?

Reply:
eh, where it is now is okay enough, some of this stuff is eventually going to get another refactor pass anyway.